environs/config/config.go

```diff
@@ -2234,4 +2279,9 @@ where possible. (default "")`,
 		Type:        environschema.Tstring,
 		Group:       environschema.EnvironGroup,
 	},
+	LoggingOutputKey: {
+		Description: `The logging output destination: database and/or syslog. (default "")`,
```

Comment:
Calling the traditional logging type database, feels like an implementation detail that user's wouldn't need to know and potentially confusing to new users. Pondering alternatives.

Reply:
My original thought was `state`